README.md

```diff
@@ -77,22 +94,114 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
+5. Copy the folder contents of the drone images folder that was just prepared onto the input named volume
+6. Create another folder for the output folders and files
+7. Run the docker container's `odm_workflow` app specifying  either the name of the shapefile or geojson file, or the URL of they [BETYdb](#betydb) instance to query for plot boundaries, and the two named volumes
+8. Copy the resulting files off the output named volume to the local folder
+9. Clean up the named volumes
+
+#### For example: <a name="opendm_can_shp_example" />
+
+You can download a sample dataset of files (archived) with names corresponding to those listed here from CyVerse using the following command.
+Be sure to replace **<username>** and **<password>** with your CyVerse username and password.
+```bash
+curl -X GET -u '<username>:<password>' https://data.cyverse.org/dav/iplant/projects/aes/cct/diag/sample-data/scif_odm_test_data.tar.gz > scif_odm_test_data.tar.gz
+gunzip scif_odm_test_data.tar.gz
+tar -xf scif_odm_test_data.tar
+```
+
+In this example we're going to assume that we're using a shapefile named `plot_shapes.shp`, that we have our drone images in a folder named `${PWD}/IMG`, and additional data in the `experiment.yaml` file.
+
+Step 1 requires creating two named Docker volumes to use when processing.
+If you already have one or more empty named volumes you can skip this step.
+You will need to have Docker running at this point.
+```bash
+docker volume create my_input
+docker volume create my_output
+``` 
+
+Step 2 involves copying the drone images into a folder:
+```bash
+cp /IMG/* "${PWD}/inputs/"
+```
+
+Step 3 copies the optional shapefile files into the same folder as the drone images:
+```bash
+cp /plot_shapes.* "${PWD}/inputs/"
+``` 
+
+In step 4 we copy the experiment.yaml file into the same folder as the drone images:
+```bash
+cp experiment.yaml "${PWD}/inputs/"
+``` 
+
+In step 5 we copy the source files onto the input named volume:
+```bash
+docker run --rm -v "${PWD}/inputs:/sources" -v my_input:/input --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp /sources/* /input/'
+``` 
+
+In step 6 we run the workflow to generate the orothomosaic image using ODM (OrthoDroneMap) and calculate plot-level canopy cover:
+```bash
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "${PWD}/inputs:/scif/data"/odm_workflow/images -v my_output:/output -e INPUT_VOLUME=my_input -e OUTPUT_VOLUME=my_output -e "INPUT_IMAGE_FOLDER=/images" -e "OUTPUT_FOLDER=/output" agdrone/canopycover-workflow:latest run odm_workflow plot_shapes.shp my_input my_output
+```
+and we wait until it's finished.
+
+In step 7 we copy the results off the named output volume to our local folder:
+```bash
+docker run --rm -v "${PWD}/outputs:/results" -v my_output:/output --entrypoint bash agdrone/canopycover-workflow:latest -c 'cp -r /output/* /results/'
+```
+The results of the processing are now in the `${PWD}/outputs` folder.
```

Comment:
What results should appear? where is the orthomosaic?

Reply:
The proposed changes broke the workflow

Reply:
after moving files from inputs/IMG/* to inputs/ in https://github.com/AgPipeline/drone-makeflow/pull/4/files#r470292286 it works